src/doctopus/files/predicates.clj

```diff
@@ -17,16 +22,18 @@
   [file-handle type-regex]
   (is-type? (.getPath file-handle) type-regex))
 
+;; ### File Predicates
+;; Used for checking that an object representing a file -- whether it's a String
+;; or a java.io.File -- has a particular extension
 
 (defn markdown?
-  "returns true only for type files."
   [file-handle]
-  (is-type? file-handle #"(?i)md|markdown|mdown$"
-            ))
-
+  (is-type? file-handle #"(?i)md|markdown|mdown$"))
```

Comment:
Not sure I grok what you mean. Like, "Is file extension a member of this vector of extensions we like"? How would that be... better?